packages/eslint-config-godaddy-minimal/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: [
+    require('./extends')('eslint-config-godaddy-react-flow')
+  ],
+  rules: {
+    'complexity': 0,
```

Comment:
I think having mechanical rules like this are a good thing. Usually "overly complex" is a legitimate concern, and it's easy enough to use a comment to make an exception in the rare case where the complexity is warranted. IMO, this is a case of a linting rule that is actually useful and not just a style preference.

Reply:
It's trivial to write overly complex code which doesn't trigger this rule anyway. Measuring cyclomatic complexity can detect a small subset of functions which are too complicated, but does little to nothing to remove responsibility from developers during the code review and refactoring processes. I agree that complex functions are usually bad; but there are cases in which functions can be comprehensible while having high cyclomatic complexity, and these are issues that should be addressed by humans, not mechanically.

Reply:
Seeing it disabled via a comment also serves as a flag during review, making discussion of why it is so complex, and if complexity can be reduced more likely.